src/components/Categories/Categories.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { useVideo } from "../../contexts/video-context";

```

Comment:
Not mandatory but good to know: You can use path alias to avoid moving back in folders using ../../
refer For Create React App without using Craco section in this https://dev.to/theodorusclarence/clean-up-your-imports-using-absolute-imports-alias-on-react-apps-next-js-cra-and-react-vite-2160

Reply:
Thanks for sharing the resource, I will look into it.